### PR TITLE
[Feature] Landing Page + Help Your Company Sections

### LINKED ISSUE #199: [Feature] Landing Page + Help Your Company Sections
### _**MERGE INTO `sponsor-us-dev-branch` and not `main`**_
Implement the sponsor us landing Page + help your company sections given [design](https://www.figma.com/file/GSDe4OZqUkyYcPSIVDI3qD/2023-Website-Designs?type=design&node-id=10-79&mode=design&t=MNPoa16UbdMjGLUt-0)

### Landing page section
Screenshot 2023-07-11 at 7 03 51 PM
### Help your company section
Screenshot 2023-07-11 at 7 05 34 PM

- Be sure to look at the rest of the main site's file structure and components you can reuse
- Be sure to link sponsib packet in the landing page section! @zhangcry should have it.

Reviewers @mike10911 @jmsargent10

### PR DESCRIPTION
**Pull request by kzeng24**
Fixes #199 

Changelist:
- Implemented desktop [design](https://www.figma.com/file/GSDe4OZqUkyYcPSIVDI3qD/2023-Website-Designs?type=design&node-id=10-79&mode=design&t=MNPoa16UbdMjGLUt-0)
- Created new folder under components/sponsor-us-section to include components for the "Sponsor Us" and "How can sponsoring HackBeanpot help your company?" sections
- Reused existing components and used ant design grid (https://ant.design/components/grid) to separate textbox into 2 columns 
- Imported images for bottom textbox
- Added onclick to "view our sponsorship packet" button such that it directs to sponsorship packet 

Note
- Used imports for animations from the Landing Page section - is that okay? Or do we have to create a copy of the animations in the sponsor-us folder?

Screenshots & Screencasts:
Screenshot 2023-07-19 at 9 54 40 AM

Screenshot 2023-07-19 at 9 55 27 AM

### CODE REVIEW

**Review by jmsargent10 — CHANGES_REQUESTED** [review]
Looks really great Kat! I have a few styling comments in regards to accuracy to the figma model, but if it doesn't need to directly mimic the figma, then I think it looks perfect!!

Changes are as follows (if necessary)
- "Sponsor Us"
     - decrease letter spacing
     - bolden
- "Recruitment" and "Market your company and product"
     - decrease letter spacing
- Other (very minor)
     - text is positioned slighly differently in relation to background (I doubt this matters esp since the new designs aren't in yet)
     - sun is rotated slightly clockwise compared to figma
     - button text looks more bold than figma
     - bolding of "celebrating innovation, diversity, and inclusion" is not as prominent as on figma
          - I know I said maybe use `<strong>` tag here but it may be nice for it to be bolder if possible

**Review by jmsargent10 — APPROVED** [review]
LGTM!! Great work on this!

File: main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <StyledWhySponsorContainer>
+      <StyledTextContainer>
+        <H3 color={colors.BUTTON_GREEN}>
+          How can sponsoring HackBeanpot help your company?
+        </H3>
+        <StyledParagraph>
+          <Row gutter={20}>
+            <Col span={12} className="gutter-row">
+              <StyledHeaderLabel>Recruitment</StyledHeaderLabel>
+              <P style={{ letterSpacing: '0.5px' }}>
```
Comment (mike10911) [inline main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx]: Please put all styles in a Styled Component

Comment (mike10911) [reply main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx]: not inline

File: main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  StyledCenterImage,
+  StyledItemContainer,
+  StyledFennecFox,
+  StyledHeaderLabel
+} from './WhySponsor.styles';
+
+import { P, H3, H4 } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import FennecFox from '../../../../shared-ui/images/fennec-fox.svg';
+import { Col, Row } from 'antd';
```
Comment (mike10911) [inline main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx]: Great job implementing this grid library! Next time you want to use a new technology run it by me or the tech channel!

File: main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+              </P>
+              <StyledItemContainer>
+                <StyledCenterImage src={Recruitment} />
+              </StyledItemContainer>
+            </Col>
+            <Col span={12} className="gutter-row">
+              <StyledHeaderLabel>
+                Market your company and product
+              </StyledHeaderLabel>
+              <P style={{ letterSpacing: '0.5px' }}>
```
Comment (mike10911) [inline main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx]: same here

**Review by mike10911 — CHANGES_REQUESTED** [review]
This is great!!! A few comments are below!
### Landing Page
- Sponsor Us should be a lot bigger to match the design below
Screenshot 2023-07-18 at 11 06 09 PM

- Let's get more right padding for the landing page blurb to reflect the design

### Help section
- Can we get more padding between the two cols to reflect the design
- A bit more padding on the left and rights side of the container to match the design
- top and bottom should have even padding
- fox component should be a bit lower to match the design

### The file structure should be:
main-site/components/sponsor-us-page/sponsor-landing-page/SponsorLandingPage.tsx
.../SponsorLandingPage.styles.ts

main-site/components/sponsor-us-page/why-sponsor-section/WhySponsorSection.tsx
.../WhySponsorSection.styles.ts

to reflect the current file structure standards

**Review by mike10911 — APPROVED** [review]
SUCH A SLAY!!! LGTM!!

**Pull request merged**